srcpkgs/audacious/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'audacious'
 #Keep in sync with audacious-plugins!
 pkgname=audacious
-version=4.0.5
+version=4.1
 revision=1
 build_style=gnu-configure
 configure_args="$(vopt_enable gtk) $(vopt_enable qt)"
```

Comment:
```suggestion
configure_args="$(vopt_enable gtk) $(vopt_enable qt) --enable-libarchive"
```
I needed to add this to build with libarchive support.

Reply:
You are correct. I added the dep and forgot the configure switch to enable it. Thank you!